test/DotNetty.Codecs.Tests/Base64Test.cs

```diff
@@ -0,0 +1,163 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace DotNetty.Codecs.Tests
+{
+    using System;
+    using System.Security.Cryptography.X509Certificates;
+    using System.Text;
+    using DotNetty.Buffers;
+    using DotNetty.Codecs.Base64;
+    using Xunit;
+
+    public class Base64Test
+    {
+        [Fact]
+        public void TestNotAddNewLineWhenEndOnLimit()
+        {
+            var src = Unpooled.CopiedBuffer(Encoding.ASCII.GetBytes("____abcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyzabcde"));
```

Comment:
>Unpooled [](start = 22, length = 8)

test with pooled might be interesting as well. E.g. it would've caught an issue with assumption that `dest.ArrayOffset == 0`